assignment/index.html

```diff
@@ -28,7 +28,10 @@
         ext: 'png'
       }).addTo(map);
 
-      L.marker([50.5, 30.5]).addTo(map);
+      L.marker([39.951011, -75.166631]).addTo(map);
+          L.marker([39.950476, -75.166191]).addTo(map);
+            L.marker([39.949259, -75.163015]).addTo(map);
```

Comment:
It's best practice to keep line spacing uniform - makes it easier to read the code.